src/features/footer/Footer.jsx

```diff
@@ -98,10 +93,16 @@ const Footer = ({ className }) => {
 
 Footer.propTypes = {
   className: PropTypes.string,
+  links: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number]))),
 };
 
 Footer.defaultProps = {
   className: '',
+  links: [
+    { label: 'Privacy Policy', url: getConfig().PRIVACY_POLICY_URL, id: 1 },
```

Comment:
Not sure if calling getConfig here, outside of the render function, could cause issues:

getConfig values can come from the build-time .env files (which wouldn't have issues here), or from the config in the OpenEDX Database, which come from an async request done during the app initialization (see the APP_READY event on the main index file). Not sure if that event would be ready when this is evaluated.

Reply:
Thanks @rodmgwgu , good point. It's probably not good practice to set default props from anything that isn't completely static. 
That being said, I tested in my local environment by building a docker image with this branch and updating `MFE_CONFIG` within the LMS and those settings were picked up correctly from the LMS through `/api/mfe_config/v1?mfe=learning-dev`. It looks like defaultProps is evaluated just before the first render() call for an element.

If this is still a concern maybe we could change Footer to a class based component and set the links in the constructor, or set the defaultProp['links'] = null and pull from the config within the render function if they're not passed in as a prop.

I pushed another possible solution, does this look better, should we stick with what we originally had?

Reply:
the solution looks good to me, thanks!